docs/containers/app-service.md

```diff
@@ -88,6 +89,22 @@ In the previous section, the image is pushed to a remote container registry. Now
 
     ![Web Application](images/app-service/webapp-homepage.png)
 
+## Deploy to ACI
+
+You can easily run your containers in Azure using [Azure Container Instances (ACI)](https://azure.microsoft.com/services/container-instances/). First, deploy the container to a container registry, such as Docker Hub or ACR, as described earlier in this article. Then, find the container in the **Registries** section of the Docker view. Use the **Refresh** button if you don't see it there. Right-click the entry for the image you want, and choose **Deploy image to Azure Container Instances**.
+
+![Deploy to ACI menu item](images/app-service/deploy-aci-menu.png)
+
+Choose **Create new ACI Context** to create a new Azure Container Instances context, give it a name, and then choose the resource group. The instance is set up and started in Azure. 
```

Comment:
I think this paragraph needs a bit of re-work/clarification.

1. You can always create a new ACI Docker context to deploy an image to ACI, but if you already have an ACI context, you can choose an existing context for deployment. So not all flows involve context creation.
1. The context you choose for deployment will be automatically activated--there is no need to activate it explicitly.